src/jquery.fancytree.js

```diff
@@ -6268,6 +6269,7 @@
 				enableAspx: 42, // TODO: this is truethy, but distinguishable from true: default will change to false in the future
 				escapeTitles: false,
 				extensions: [],
+				copyFunctionsToData: false,
```

Comment:
Let's keep the sort order (even if it is not completely perfect yet, I will fix those later ;-)

Reply:
What do you mean?

Reply:
no worries, I will sort